invisible_cities/cities/components.py

```diff
@@ -1002,3 +1037,267 @@ def calculate_and_save_buffers(buffer_length    : float        ,
     # Filter out order_sensors if it is not set
     buffer_definition = pipe(*filter(None, find_signal_and_write_buffers))
     return buffer_definition
+
+def copy_E_or_Ec_to_Ep(energy_type: HitEnergy):
```

Comment:
As  @gonzaponte  suggest, to keep our naming convention, this function can be called 
`Efield_copier` and the inner one `copy_Efield` (this later name can also be used in the pipe elements namings)

Reply:
It sounds good, I've changed that.